cmd/neofs-cli/internal/client/sdk.go

```diff
@@ -9,8 +9,31 @@ import (
 	"github.com/nspcc-dev/neofs-sdk-go/client"
 )
 
+// NetAddr groups information about network endpoint needed for client
+// connection.
+type NetAddr struct {
+	tlsEnabled bool
+
+	uri string
+}
+
```

Comment:
What about having `HostAddrWithScheme` or similar on `network.Address`. Maybe even a function.

Reply:
I suggest to postpone the proposed refactor since current PR is a hotfix. See https://github.com/nspcc-dev/neofs-node/issues/1354.

Reply:
That was also needed to fix similar problem in Storage node app, so done in 629150ea44bbb71455e2ea611fdcb7ec4e206702.